test/nools/lib.tasks.spec.js

```diff
@@ -166,14 +234,33 @@ describe('nools lib', function() {
         config.tasks[0].events = [event, event];
 
         // when
-        const emitted = runNoolsLib(config).emitted;
+        const { emitted } = runNoolsLib(config);
 
         // then
         expect(emitted).to.have.property('length', 4);
         expectAllToHaveUniqueIds(emitted);
       });
 
-      it('dueDate function is invoked with expected data', function() {
+      it('given contact without reported_date, dueDate defaults to now', () => {
+        sinon.useFakeTimers(1);
+
+        // given
+        const config = {
+          c: personWithReports(aReport()),
+          targets: [],
+          tasks: [ aPersonBasedTask() ],
+        };
+        delete config.c.contact.reported_date;
+
+        // when
+        const { emitted } = runNoolsLib(config);
+
+        // then
+        expect(emitted[0]).to.nested.include({ 'contact._id': 'c-2' });
+        expect(emitted[0].date.getTime()).to.be.lt(24 * 60 * 60 * 1000);
```

Comment:
This is just a vicious hack because I hate the Date class. Open to suggestions for handling this more precisely. I tried timezoneOffset without success.

Reply:
Yea, maybe you could actually generate the date that you expect and use that? Like:
```
const expectedDate = new Date();
expectedDate.setHours(0, 0, 0, 0);
expect(emitted[0].date.getTime()).to.equal(expectedDate.getTime());
```

Reply:
You're great.